src/FSharp.Data.GraphQL.Shared/TypeSystem.fs

```diff
@@ -878,7 +878,9 @@ and ResolveFieldContext =
       /// parametrized inputs.
       Args : Map<string, obj>
       /// Variables provided by the operation caller.
-      Variables : Map<string, obj> }
+      Variables : Map<string, obj>
+      /// Field path
+      Path : obj list }
```

Comment:
Hi @mickhansen 
Since this is consumable by end users, can we make this a `string` rather than a `obj list`.

Reply:
@jberzy The error type expects an `obj list`: https://github.com/fsprojects/FSharp.Data.GraphQL/blob/3cd0940df3150be671c284d205f9071a5d5950ed/src/FSharp.Data.GraphQL.Server/Execution.fs#L16 which was my primary intended target for this PR.